examples/browser-to-browser/README.md

```diff
@@ -0,0 +1,57 @@
+# js-libp2p-webrtc Browser to Browser
+
+This example leverages the [vite bundler](https://vitejs.dev/) to compile and serve the libp2p code in the browser. You can use other bundlers such as Webpack, but we will not be covering them here.
+
+## Running the Relay Server
+
+For browsers to communicate, we first need to run the LibP2P relay server:
+
+```shell
+npm run relay
+```
+
+Copy one of the multiaddresses in the output.
+
+## Running the Example
+
```

Comment:
In order to run this example, the `@libp2p/webrtc` package must be built as well, so before running `npm run start` here, one must run `npm run build`  in the root directory then `cd` into `examples/browser-to-browser` and then they can  run 
```shell 
npm i && npm run start
```